src/components/Spinner.tsx

```diff
@@ -0,0 +1,42 @@
+import clsx from 'clsx'
+import React, { FC } from 'react'
+
+type Props = {
+  color?: string
+  size?: string
+}
+
+const Spinner: FC<Props> = ({ color, size }) => {
```

Comment:
nit: since [this conversation](https://github.com/facebook/create-react-app/pull/8177) I am personally not using `FC<T>` generic anymore. Seem it adds little/no-value.